notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1191 @@
+{
```

Comment:
<p>Do you think you could improve you results, for accuracy , briair score, roc, and the reliability diagram if you resampled your training data to be balanced? Currently it is 96.7% pass and 3.2% fail. So according to the accuracy score in both cases this is modeling the distribution of the labels and not exhibiting much predictive power.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Yes, I can try that out and see if that gives us better results. 👍

Reply:
:+1:

Reply:
We tried various resampling techniques and by undersampling, we achieved better results for the three metrics we were calculating, so we will add an updated notebook with resampled data. Thanks for the suggestion @MichaelClifford